cumulus/parachains/runtimes/testing/penpal/src/xcm_config.rs

```diff
@@ -275,25 +277,41 @@ where
 	}
 }
 
+// This asset can be added to AH as Asset and reserved transfer between Penpal and AH
+pub const RESERVABLE_ASSET_ID: u32 = 1;
 // This asset can be added to AH as ForeignAsset and teleported between Penpal and AH
 pub const TELEPORTABLE_ASSET_ID: u32 = 2;
+
+pub const ASSETS_PALLET_ID: u8 = 50;
+pub const ASSET_HUB_ID: u32 = 1000;
```

Comment:
I would turn these constants as `pub storage` with default values, to be able change them possibly with `System::set_storage`. We had similar problem for reusing penpal with Kusama/Polkadot XCM emulator tests: https://github.com/paritytech/polkadot-sdk/pull/3564

Reply:
But are those values really needed to be configurable? Could you give an example?

Reply:
Right, we don't need that now. Fortunately, 1000 and 50 works for the polkadot-sdk and fellows.